library/hawkular_alerts_group_trigger.py

```diff
@@ -260,6 +326,7 @@ def main():
     enabled    = module.params['enabled']
     verify_ssl = module.params['verify_ssl']
     ca_file    = module.params['ca_file_path']
+    conditions = module.params['group_conditions']
```

Comment:
I needed a different name for the dict and the condition list within the dict. If you have a better naming option let me know :)